docs/en/Configuring-Alluxio-with-Azure-Blob-Store.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: global
+title: Configuring Alluxio with Azure Blob Store
+nickname: Alluxio with Azure Blob Store
+group: Under Store
+priority: 0
+---
+
+* Table of Contents
+{:toc}
+
+This guide describes how to configure Alluxio with [Azure Blob Store](https://azure.microsoft.com/en-in/services/storage/blobs/) as the under storage system.
+
+## Initial Setup
+
+To run an Alluxio cluster on a set of machines, you must deploy Alluxio binaries to each of these machines.You can either [compile the binaries from Alluxio source code](http://alluxio.org/documentation/master/Building-Alluxio-Master-Branch.html), or [download the precompiled binaries directly](http://alluxio.org/documentation/master/Running-Alluxio-Locally.html).
+
+Then, if you haven't already done so, create your configuration file with `bootstrapConf` command.
+For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_HOSTNAME` should be set to `localhost`
+
+```
+$ ./bin/alluxio bootstrapConf <ALLUXIO_MASTER_HOSTNAME>
+```
+
+Alternatively, you can also create the configuration file from the template and set the contents manually.
+
+```
+$ cp conf/alluxio-env.sh.template conf/alluxio-env.sh
+```
+
+Also, in preparation for using Azure blob store with alluxio, create a new container in your azure storage account or use an existing container. You should also note that the directory you want to use in that container, either by creating a new directory in the container, or using an existing one. For the purposes of this guide, the Azure storage account name is called AZURE_ACCOUNT, the container in that storage account is called AZURE_CONTAINER and the directory in that bucket is called AZURE_DIRECTORY. For more information about Azure storage account, Please see [here](https://docs.microsoft.com/en-us/azure/storage/storage-create-storage-account)
+
+## Configuring Alluxio
+
+Alluxio doesn't directly support blob store as its underfs. We need to use Alluxio via the HDFS interface and by configuring hadoop to interact with Azure blob store. You can find more about running Hadoop on Azure Blob Store over [here](http://hadoop.apache.org/docs/r2.7.1/hadoop-azure/index.html).
```

Comment:
Instead of these sentences it would be more clear if it were something like: "Alluxio can support the Azure blob store via the HDFS interface. You can find more about ..."